### PR TITLE
Add barcode layouts for some Year 3 projects

### PR DESCRIPTION
**Pull request by jccraft-uw**
Add the barcode layouts for the Apple respiratory study, the public schools
study, and the Adult Family Home outbreak / Workplace outbreak projects.

### CODE REVIEW

**Comment by jccraft-uw** [comment]
[Create collections & identifier set for new Y3 studies](https://trello.com/c/FCPymnm0/963-create-collections-identifier-set-for-new-y3-studies)

File: lib/id3c/labelmaker.py [hunk 1]
```diff
@@ -302,6 +330,10 @@ class CollectionsChildcareLayout(LabelLayout):
     'collections-uw-observed': CollectionsUWObservedLayout,
     'collections-uw-home': CollectionsUWHomeLayout,
     'collections-childcare': CollectionsChildcareLayout,
+    'collections-school-testing': CollectionsSchoolTesting,
+    'collections-apple-respiratory': CollectionsAppleRespiratory,
+    'collections-adult-family-home-outbreak': CollectionsAdultFamilyHomeOutbreak,
+    'collections-workplace-outbreak': CollectionsWorkplaceOutbreak,
```
Comment (kschwabefry) [inline lib/id3c/labelmaker.py:336]: By convention, would be nice to append Layout to these variables name, (e.g. `CollectionsSchoolTestingLayout`,  `CollectionsAppleRespiratoryLayout`)

**Review by kschwabefry — CHANGES_REQUESTED** [review]

File: lib/id3c/labelmaker.py (hunk 1 above)
Comment (jccraft-uw) [reply lib/id3c/labelmaker.py]: Yes, indeed. Fixed.

**Review by kschwabefry — APPROVED** [review]

**Comment by jccraft-uw** [comment]
Deployed.

**Pull request merged**